mappings/net/minecraft/client/gui/screen/CreditsScreen.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS net/minecraft/class_445 net/minecraft/client/gui/screen/CreditsScreen
+	FIELD field_24260 COLORS Ljava/lang/String;
```

Comment:
```suggestion
	FIELD field_24260 OBFUSCATE_PLACEHOLDER Ljava/lang/String;
```

Correct me if I'm wrong, but it's more like a placeholder rather than a mere collection of colors.

Reply:
It replaces the color with white obfuscation if it is not in the `COLORS` field.

Reply:
I can't find the code that checks if something is in the `COLORS` field. Only I found is `String#indexOf` taking the field **as an argument** and this means it searches for the field in the other string.

Reply:
It uses a string rather than a collection for the colors whitelist.

Reply:
Fixed in 1f1569f25302c22c08bcd8300e497b4e2e3d433e